cmd/sink/filters/filters.go

```diff
@@ -25,14 +25,17 @@ import (
 )
 
 const (
-	globalKeyEnvVar = "KUBEARCHIVE_NAMESPACE"
 	filtersCmName   = "sink-filters"
+	GlobalNamespace = "___global___"
```

Comment:
I think it might be better to import this constant from the operator so that we don't end up in a situation where the values in the sink and the operator get out of sync

Reply:
at the same time, the SinkFiltersConfigMapName could also be imported from the operator to remove duplication of constants as well and would allow us to remove `filtersCmName` from this file

Reply:
Done. I did the import the other way, so we keep the value in the filters code.